fastlane/lanes/release.rb

```diff
@@ -457,14 +504,19 @@
 # @param [Boolean] beta Indicate if we should build a beta or regular release
 #
 def trigger_buildkite_release_build(branch:, beta:)
-  buildkite_trigger_build(
+  build_url = buildkite_trigger_build(
     buildkite_organization: BUILDKITE_ORGANIZATION,
     buildkite_pipeline: BUILDKITE_PIPELINE,
     branch: branch,
     environment: { BETA_RELEASE: beta },
     pipeline_file: 'release-builds.yml',
     message: beta ? 'Beta Builds' : 'Release Builds'
   )
+
+  return unless is_ci
+  
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/wordpress-mobile/WordPress-iOS/blob/b318b700f75303b75dbea0608be7c6cf0f260036/fastlane/lanes/release.rb#L517"></span>Layout/TrailingWhitespace: Trailing whitespace detected.</td>
    </tr>
  </tbody>
</table>

Reply:
🤔 where is the trailing whitespace RuboCop / Danger detects?

<img width="574" alt="image" src="https://github.com/user-attachments/assets/facdea91-f0f6-479d-9975-51fa41c327d3">

Reply:
It was on line 517, even though the GitHub diff didn't show it

<img width="713" alt="image" src="https://github.com/user-attachments/assets/b195aa58-6340-466b-8ba0-b277928bbe2d">

536b1c0